src/lang/expr.cc

```diff
@@ -47,6 +47,10 @@ std::ostream& operator<<(std::ostream& os, const NodeRef& n) {  // NOLINT(*)
   return os;
 }
 
+void DumpNode(const NodeRef& n) {  // NOLINT(*)
+  std::cout << n << "\n";
```

Comment:
why "\n", std::endl is better?

Reply:
```std::endl``` forces flushing the output buffer. Unless we really want to do that, ```"\n"``` is actually slightly preferred, e.g.:

    https://llvm.org/docs/CodingStandards.html#avoid-std-endl

Although it's from another code standard, I think it would apply here, too. For our debugging purpose here, it doesn't seem to have much difference in term of using ```std::endl``` or ```"\n"```.